compute/cloud-client/src/main/java/compute/snapshotschedule/CreateSnapshotSchedule.java

```diff
@@ -51,7 +53,7 @@ public static void main(String[] args)
     // https://cloud.google.com/compute/docs/disks/snapshots?authuser=0#selecting_a_storage_location
     String storageLocation = "US";
     // Determines what happens to your snapshots if the source disk is deleted.
-    String onSourceDiskDelete = "KEEP_AUTO_SNAPSHOTS";
+    String onSourceDiskDelete = KEEP_AUTO_SNAPSHOTS.toString();
```

Comment:
Remove this variable. Inline the enum directly in line 81

Reply:
Thank you! Fixed